cms.php

```diff
@@ -1,4 +1,35 @@
 <?php
+session_start ();
+
+if (!empty($_POST ['user-name'] && $_POST ['password'])) {
+
+    $db = new PDO('mysql:host=127.0.0.1; dbname=jackdb', 'root');
+
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE,
+        PDO::FETCH_ASSOC);
+
+    $query = $db->prepare("SELECT `password` , `user` 
+    FROM `users` WHERE `user` LIKE CONCAT('%', :user, '%');");
+
+    $query->bindParam(':user', $_POST ['user-name']);
+
+    $query->execute();
+    $out = $query->fetchAll();
```

Comment:
You're pulling out one row so I'd use `fetch()` rather than `fetchAll()`

Reply:
Good idea. I've also changed the arrays to match.